build.json

```diff
@@ -6,12 +6,12 @@
     "1.1.2"
   ],
   "LegacyDotNetVersion": "1.0.0-preview2-1-003177",
-  "RequiredMonoVersion": "5.2.0.196",
+  "RequiredMonoVersion": "5.8.0.129",
```

Comment:
Let's not change this. The latest Mono package works fine when 5.2.0.196 is installed.

Reply:
OK, that's great. so let's merge https://github.com/OmniSharp/omnisharp-roslyn/pull/1137 and then I'll rebase this and get rid of these changes